cloud-sql/sql-server/sqlalchemy/Dockerfile

```
@@ -0,0 +1,45 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.8-buster
+
+#Download the desired package(s) for Microsoft ODBC 17 Driver
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+RUN curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list
+RUN apt-get update
+RUN ACCEPT_EULA=Y apt-get install -y msodbcsql17
+RUN apt-get install unixodbc-dev
```

nit: Let's chain these together into a single layer:
```suggestion
RUN (curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add) - && \
         curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
         apt-get update && ACCEPT_EULA=Y apt-get install -y msodbcsql17  unixodbc-dev
```